src/test/java/com/activestate/cloudfoundryjenkins/ManifestReaderTest.java

```diff
@@ -30,6 +30,19 @@ public void testGetApplicationInfo() throws Exception {
         assertEquals("hello-java", result.get("name"));
         assertEquals("512M", result.get("memory"));
         assertEquals("target/hello-java-1.0.war", result.get("path"));
+        assertEquals(null, result.get("null"));
```

Comment:
Yeah because the manifest reader reads the file and it is read in as null.  The DeploymentInfo class sets the default.   I guess I just like going above and beyond with tests....